packages/blocks/src/widgets/drag-handle/index.ts

```diff
@@ -0,0 +1,580 @@
+import type {
```

Comment:
Is it possible to remove the legacy `blocks/src/components/drag-handle.ts` for now? Regressions are acceptable here.

Reply:
I will verify whether the new drag handle can work properly under edgeless mode, if it works well, I will remove the legacy drag-handle component